### ISSUE TITLE
Github workflow permission changes and automation

### ISSUE THREAD

**Issue by sarnold** [PennyDreadfulMTG/pystache#13]
Although The Right Thing to do, the github workflow changes to default token/repository permissions has been kinda painful, and I'm still working through various things.  That said, I'm not sure what's up with your badge error just yet, but I just finished some workflow updates to get things working again in a couple of "org" scenarios (some of which involved adding a simple org-level bot to generate a usable token for different automation chores).

For the coverage workflow in VCTLabs org, it ended up being action upgrades and a regex fix to get it working for in-repo pull requests (but not external forks).  The full update still needs the split workflows/artifacts-passing refactor :/

Some info on the various auth solutions: https://github.com/peter-evans/create-pull-request/blob/main/docs/concepts-guidelines.md#authenticating-with-github-app-generated-tokens

**Comment by bakert** [comment]
What are the GitHub actions that require tokens? I think you've lost me here :)

**Comment by silasary** [comment]
Hmm.  Changing the access token didn't appear to work. Or do anything at all?

**Comment by sarnold** [comment]
Sorry, RL-intervention again, plus I have some pending test driver updates to push a PR for (nose isn't really maintained and pypi nose will break on Python 3.10).  Anyway, I need to ask what all we did in VCTLabs org to get coverage workflow "working" again, mainly because someone else added the simple "access" bot to go with these token changes.  I'll need to make another PR for the workflow updates, hopefully this weekend.

On the flip-side I fixed/cleaned up my `gitchangelog-action` this week so I'm now an "official" consumer of the new src package on pypi :beers:
